### ISSUE TITLE
Allow installation with plugin manager

### ISSUE THREAD

**Issue by mitchelljkotler** [el1t/statusline#8]
Instead of a custom script, what if we could install with antigen/zgen/zplug?

**Comment by el1t** [comment]
I agree, that would be very convenient. I don't know how to optimize it for plugin loaders, but I think installation is possible:
```sh
antigen theme el1t/statusline oh-my-zsh/statusline
```
should work?
